asciidoc-confluence-publisher-maven-plugin/src/main/java/org/sahli/asciidoc/confluence/publisher/maven/plugin/AsciidocConfluencePublisherMojo.java

```diff
@@ -47,61 +47,63 @@
 @Mojo(name = "publish")
 public class AsciidocConfluencePublisherMojo extends AbstractMojo {
 
+    static final String PREFIX = "asciidoctor.confluence.";
+
     @Parameter(defaultValue = "${project.build.directory}/asciidoc-confluence-publisher", readonly = true)
     private File confluencePublisherBuildFolder;
 
-    @Parameter
+    @Parameter(property = PREFIX + "rootFolder")
```

Comment:
I prefer to use `asciidocRootFolder` to be inline with the documented configuration property names.

Reply:
I can change that.